src/components/UnderConstruction.js

```diff
@@ -6,7 +6,7 @@ import { WarningAlt } from "@carbon/icons-react";
  */
 const UnderConstruction = () => {
   return (
-    <div>
+    <div className="underConstruction">
```

Comment:
can we add center alignment to the under construction pages?
<img width="506" alt="Screen Shot 2023-03-17 at 11 36 38 AM" src="https://user-images.githubusercontent.com/11401353/225990165-bfd9dfe8-d38f-4c5a-a6d6-f7ffbcb19491.png">

Reply:
idk how this got messed up, but fixed